playground/mocks/data/idp/idx/authenticator-enroll-password.json

```diff
@@ -21,7 +21,8 @@
                 {
                   "name": "passcode",
                   "label": "Enter password",
-                  "secret": true
+                  "secret": true,
+                  "required": true
```

Comment:
Can you double-check to ensure the `required` field appears here rather than a level up between line 18 and 19? I have seen it appear at both levels.

Reply:
Oh yea, looks like it should technically be at the `credentials` level. 

![image](https://user-images.githubusercontent.com/97472729/202032639-7472ddd2-10c1-400f-a2e0-2ed1ef4950c5.png)